### PR TITLE
Join different Tables In Vet Clinic

### PR DESCRIPTION
**Pull request by Atril33**
## **Vet Clinic**
_**In this Project, I have done a few suggested requirements**_
Created a _Database_ by the name of `vet_clinic.`
Created a _Table_ by the name of `animals`.
Added new data in _Table_.
Used _UPDATE, DELETE, COUNT, AVG, & GROUP BY_ in this **Vet_Clinic**.
Used _JOIN_ in this **Vet_Clinic**.

**Project requirements**
You Can Find This Project Requirements On This [Page ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md)

PREVIEW OF _Who was the last animal seen by William Tatcher?_

PREVIEW OF _How many different animals did Stephanie Mendez see?_

PREVIEW OF _List all vets and their specialties, including vets with no specialties._

PREVIEW OF _List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020._

PREVIEW OF _What animal has the most visits to vets?_

PREVIEW OF _Who was Maisy Smith's first visit?_

PREVIEW OF _Details for most recent visit: animal information, vet information, and date of visit._

PREVIEW OF _How many visits were with a vet that did not specialize in that animal's species?_

PREVIEW OF _What specialty should Maisy Smith consider getting? Look for the species she gets the most._

_Looking forward to your review._

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -62,3 +62,46 @@ VALUES
...
+/* Day 4. */
+
+INSERT INTO vets (id, name, age, date_of_graduation) 
+   VALUES 
+  ('William Tatcher', 45, '2000-04-23'),
+  ('Maisy Smith', 26, '2019-01-17'),
+  ('Stephanie Mendez', 64, '1981-05-04'),
+  ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '2000-04-23');
```
Comment (rotimiazeez) [inline data.sql:75]: I think you forgot to add the data for specialization to your data.sql file and instead you have data for vets twice. It happens to the best of us😅 kindly add the specialization data to fulfill the requirements of this milestone😉
Screenshot_20230520_175024

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Atril33  👍🏽,
## Good job so far!
### To Highlight 👇🏽
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: queries.sql [hunk 2]
```diff
@@ -40,13 +40,66 @@ SELECT MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
...
 SELECT animals.name, owners.full_name FROM owners INNER JOIN animals ON animals.id = owners.id;
 SELECT animals.species_id, COUNT(animals.id) AS total_animals FROM animals INNER JOIN species ON animals.id = species.id GROUP BY animals.species_id;
 SELECT owners.full_name AS name, animals.name AS Animal_name, animals.species_id AS Animal_species FROM animals JOIN owners ON animals.id = owners.id WHERE species_id = 'Digimon' AND full_name = 'Jennifer Orwell';
 SELECT owners.full_name As name, animals.name AS animal_name FROM animals JOIN owners ON animals.id = owners.id WHERE full_name = 'Dean Winchester ' AND escape_attempts = 0;
 SELECT owners.full_name, COUNT(owners.age) AS total_animals FROM owners JOIN animals ON owners.id = animals.id GROUP BY full_name ORDER BY total_animals DESC LIMIT 1; 
+
+/* Day 4. */
+
+ SELECT animals.name AS Animal_names, vets.name AS Name, visits.date_of_visit AS Last_Visits FROM animals, vets, visits WHERE vets.name = 'William Tatcher' AND vets.id = visits.animals_id ORDER BY visits.date_of_visit DESC
+LIMIT 1;
```
Comment (Zuhaib042) [inline queries.sql]: - [ ] The last animal visited by William Tatcher is `Blossom` but in your screenshot its Agumon so consider fixing it.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @Atril33 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✅ Descriptive PR
✅ Professional Readme
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Atril33, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive `PR` summary

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

#### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**